diagnostics/model_test/verify_model.py

```diff
@@ -0,0 +1,161 @@
+import tensorflow as tf
+import argparse
+import numpy as np
+import ngraph
+import json
+import os
+
+
+def calculate_output(param_dict, select_device, input_example):
+    """Calculate the output of the imported frozen graph given the input.
+
+    Load the graph def from frozen_graph_file on selected device, then get the tensors based on the input and output name from the graph,
+    then feed the input_example to the graph and retrieves the output vector.
+
+    Args:
+    param_dict: The dictionary contains all the user-input data in the json file.
+    select_device: "NGRAPH" or "CPU".
+    input_example: Random generated input or actual image.
+
+    Returns:
+        The output vector obtained from running the input_example through the graph.
+    """
+    frozen_graph_filename = param_dict["frozen_graph_location"]
+    input_tensor_name = param_dict["input_tensor_name"]
+    output_tensor_name = param_dict["output_tensor_name"]
+
+    with tf.gfile.GFile(frozen_graph_filename, "rb") as f:
+        graph_def = tf.GraphDef()
+        graph_def.ParseFromString(f.read())
+
+    with tf.Graph().as_default() as graph:
+        with tf.device('/job:localhost/replica:0/task:0/device:' + select_device + ':0'):
+            tf.import_graph_def(graph_def)
+
```

Comment:
tf.device setting is not required now. In C++ we use ActivateNgraph, DeactivateNgraph. In python need to figure it out?

Reply:
Oh yes, we need to discuss with @avijit-nervana , since I remember previously Chris is doing some python functions to deactivate ngraph in python files. We can check the status of this.